components/scream/data/scream_input.yaml

```diff
@@ -18,4 +18,118 @@ Atmosphere Driver:
     Reference Grid: Physics GLL
     Dynamics Driven:
       Dynamics Namelist File Name: ./data/theta.nl
+
+SCREAM:
+
+  #=====================
+  # I'm confused how we should set this. You shouldn't be able
+  # to change this once a run starts, but you should be able to choose it
+  # when you start a new run... What if your initial condition is for year 2000
+  # but you're doing a control run and want it to start at 01-01-0001?
+  Start Date: 01-01-2000
+
+  # Model Timesteps
+  whole_model_dt: 3600.0 #aka phys/dyn coupling dt
+
+  # Number of sub or supersteps of whole_model_dt for given proc.
+  # maybe have positive #s be supersteps and negative numbers being
+  # number of substeps? This might require a generalization of substepping
+  # since everything except rad always substeps and rad always supersteps?
+  # There's also an issue here where AMB just
+  # rejiggered dyn's substepping to be ~resolution invariant. Maybe
+  # use that impl for all substeps?
+  n_turb_steps: 1
+  n_micro_steps: 1
+  n_rad_steps: 1
+  dt_remap_factor: 2
+  dt_tracer_factor:  6
+
+  Tuning Parameters:
+    SHOC:
+      Ckh_s_def: 0.1 #using "moderate-stable"
+      Ckm_s_def: 0.1 #vals here as used by
+      Ck_s_min: 0.1  #DYAMOND2 SCREAMv0 run
+      #there will definitely eventually be more
+
+    P3:
+      mu_r_constant: 1.0
+      p3_qc_accret_expon: 1.15
+      p3_qc_autocon_expon: 2.47
+      #there will definitely eventually be more
+
+  Well-Mixed Greenhouse Gases:
+    co2vmr: 367.000000e-6
+    ch4vmr: 1751.022e-9
+    n2ovmr: 315.85e-9
+    f11vmr: 676.0526e-12
+    f12vmr: 537.05e-12
+
+  Code Options:
+    include_subgrid_variability: true
+    enable_T_warnings: true
+
+  Input Files:
+    input_root                  : default # this will default to DIN_LOC_ROOT for the current machine
+
+    # The filepaths below will be prepended with $input_root unless the path looks absolute
+    atm_ic_file                 : atm/cam/inic/homme/cami_mam3_Linoz_ne4np4_L72_c160909.nc
+    topog_file                  : atm/cam/topo/USGS-gtopo30_ne4np4pg2_16x_converted.c20200527.nc
+    #SPA has a different file for each month of the year, so just give directory
+    spa_file_dir                : atm/cam/rad/
+    ice_optics_file             : atm/cam/physprops/iceoptics_c080917.nc
+    liq_optics_file             : atm/cam/physprops/F_nwvl200_mu20_lam50_res64_t298_c080428.nc
+    rrtmgp_coefficients_file_lw : atm/cam/rad/rrtmgp_coefficients_lw_20181204.nc
+    rrtmgp_coefficients_file_sw : atm/cam/rad/rrtmgp_coefficients_sw_20181204.nc
+    solar_data_file             : atm/cam/solar/spectral_irradiance_Lean_1976-2007_ave_c20160517.nc
+    micro_p3_lookup_dir         : atm/cam/physprops
+
+    solar_data_ymd : 20000101
+    solar_data_forcing : constant
+
+    micro_p3_tableversion : 4
+
+  Output Files:
```

Comment:
Maybe we should default to "None" as an option and if the output manager detects "none" then it skips everything else.  At the moment, it will likely throw an error because there isn't any viable information listed in the "Output Files" category.

Reply:
@AaronDonahue , I'm not a YAML expert, but I think this will work fine. When a client asks for Output Files, they will get an empty map/dictionary.

Reply:
Whoops, I just checked and the python version actually defaults to None: `('Output Files', None)`, so the behavior already matches what you want.

Reply:
Okay great, then I will need to put in a hot fix to the output manager once this PR goes in.  I'll make a git-issue.  It shouldn't impact any of the unit tests since the ones that use I/O set Output Files.

Reply:
We should adjust the comment that follows to something like.
"This should be a bracketed list of all the output control yaml files"
On the same note, I think we will eventually have a default yaml output control file for CIME runs that we will need to copy over to the run directory when the case is built.  We may need to think about this in a little more depth, but that conversation can wait until we have a CIME run working.